modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss

```diff
@@ -5,6 +5,27 @@ $tab-header-height: 38px;
 
 .global-apps-menu-modal {
 	.global-apps-menu-header {
+		.autofit-row {
+			@include media-breakpoint-down(xs) {
```

Comment:
I don't love the `media-breakpoint-down` but I guess it was necessary and "cheaper" than `media-breakpoint-up` 😁

Reply:
yeah, I've tried to do it by default helpers but is not possible, it is a corner case so I think the best is a custom CSS